draft-ietf-tls-tls13.md

```diff
@@ -2649,16 +2651,28 @@ Structure of this message:
        } Certificate;
 
 certificate_list
-: This is a sequence (chain) of certificates.  The server's
-  certificate MUST come first in the list.  Each following
-  certificate MUST directly certify the one preceding it.  Because
-  certificate validation requires that root keys be distributed
-  independently, the self-signed certificate that specifies the root
-  certification authority (CA) MAY be omitted from the chain, under the
-  assumption that the client must already possess it in order to
-  validate it in any case.
+: This is a sequence (chain) of certificates. The sender's
+  certificate MUST come first in the list. Each following
+  certificate SHOULD directly certify one preceding it. Because
+  certificate validation requires that trust anchors be distributed
+  independently, a self-signed certificate that specifies a
+  trust anchor MAY be omitted from the chain, provided that
+  supported peers are known to possess any omitted certificates
+  they may require.
 {:br }
 
+Note: Prior to TLS 1.3, "certificate_list" ordering was required to be strict,
+however some implementations already allowed for some flexibility. For maximum
+compatibility, all implementations SHOULD be prepared to handle potentially
+extraneous certificates and arbitrary orderings from any TLS version (with
+the exception of the sender's certificate). Some servers are configured to send
+both a current and deprecated intermediate for transitional purposes, and others
+are simply configured incorrectly, but these cases can nonetheless be validated
+properly by clients capable of doing so. Whilst the chain MAY be ordered in a
+variety of ways, the peer's end-entity certificate MUST be the first element in
+the vector, and implementations MUST abort with a fatal "bad_certificate" alert
+in the event that this is not the case.
```

Comment:
It's not clear that this is actually actionable just because of how implementations behave.